instagram/src/Components/PostContainer/PostPage.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import dummyData from "../../dummy-data";
+import PostContainer from "./PostContainer";
+
+class PostPage extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      postList: [],
+      commentField: ""
+    };
+  }
+  componentDidMount() {
+    this.setState({ postList: dummyData });
+  }
+
+  updateCommentField = event => {
+    this.setState({ [event.target.name]: event.target.value });
+  };
+
+  submitNewComment = (event, timestamp) => {
+    event.preventDefault();
+    const newPostList = this.state.postList.map(eachPost => {
+      if (eachPost.timestamp === timestamp) {
+        eachPost.comments.push({
+          username: "Clark Kent",
```

Comment:
i think you should try replacing the hardcoded `username: 'Clark Kent' with  the username on localStorage